src/geopm_topo.h

```diff
@@ -85,10 +85,22 @@ enum geopm_domain_e {
      *        on-package graphics)
      */
     GEOPM_DOMAIN_PACKAGE_ACCELERATOR = 9,
+    /*!
+     * @brief Accelerator card subdevices on the PCI Bus (e.g
+     *        Level Zero subdevices)
+     */
+    GEOPM_DOMAIN_BOARD_ACCELERATOR_SUBDEVICE = 10,
     /*!
      * @brief Number of valid domains.
      */
-    GEOPM_NUM_DOMAIN = 10,
+    GEOPM_NUM_DOMAIN = 11,
+};
+
+enum geopm_levelzero_domain_e {
+    GEOPM_LEVELZERO_DOMAIN_ALL = 0,
+    GEOPM_LEVELZERO_DOMAIN_COMPUTE = 1,
+    GEOPM_LEVELZERO_DOMAIN_MEMORY = 2,
+    GEOPM_LEVELZERO_DOMAIN_SIZE = 3
 };
```

Comment:
GEOPM_LEVELZERO_DOMAIN may not need to be part of geopm_topo.h.  It may be reasonable to move this to an equivalent geopm_accelerator_topo.h for accelerators.